dolphinscheduler-e2e/dolphinscheduler-e2e-core/src/main/java/org/apache/dolphinscheduler/e2e/core/DolphinSchedulerExtension.java

```diff
@@ -73,36 +74,27 @@
     private HostAndPort address;
     private String rootPath;
 
+    private DockerImageName imageName;
+
+    private Path record;
+
+
     @Override
     @SuppressWarnings("UnstableApiUsage")
     public void beforeAll(ExtensionContext context) throws IOException {
         Awaitility.setDefaultTimeout(Duration.ofSeconds(60));
         Awaitility.setDefaultPollInterval(Duration.ofSeconds(10));
 
+        getRecordPath();
+
         if (LOCAL_MODE) {
             runInLocal();
         } else {
             runInDockerContainer(context);
         }
 
-        final Path record;
-        if (!Strings.isNullOrEmpty(System.getenv("RECORDING_PATH"))) {
-            record = Paths.get(System.getenv("RECORDING_PATH"));
-            if (!record.toFile().exists()) {
-                if (!record.toFile().mkdir()) {
-                    throw new IOException("Failed to create recording directory: " + record.toAbsolutePath());
-                }
-            }
-        } else {
-            record = Files.createTempDirectory("record-");
-        }
+        getBrowserContainerByOsName();
```

Comment:
`getXxxx` usually returns an object and is assigned to some variable, but your  `getXxxx` instead, sets a field, which I don't think is a good practice.

Please either change the method name to `setXxx` or make it return the object and assign it to the field

Reply:
I'll fix it.